### PR TITLE
Item id holding

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package com.epam.reportportal.junit5.utils;
```
Comment (HardNorth) [inline src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java]: Copyright

File: src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import com.epam.reportportal.service.tree.TestItemTree;
+import io.reactivex.annotations.Nullable;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
```
Comment (HardNorth) [inline src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java]: Please add a description of the class

Comment (Yumfriez) [reply src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java]: provided

Comment (Yumfriez) [reply src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java]: provided

File: src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/reportportal/junit5/utils/ItemTreeUtils.java]: fixed

**Pull request merged**